### DISCUSSION TITLE
Script: how to set output panel to the "Messages" page?

### DISCUSSION THREAD

**Discussion by bryango** [texstudio-org/texstudio#2193] [Q&A] [answered]
From the examples I learn how to trigger the outputview in a script:
```js
app.getManagedAction("main/view/show/outputview").trigger()
```
However, what I really want is for it to show the "Messages" page. I search through the source and it seems that this is achieved by `setCurrentPage(MESSAGES_PAGE)`; see e.g.
https://github.com/texstudio-org/texstudio/blob/26c5da4797557557a2d4b0f0aed5e8cfc9728006/src/toolwidgets.cpp#L359-L363
Can I achieve this using a macro / script? Thanks!

**Comment by sunderme** [comment, marked as answer]
It looks like it is not possible through script.

**Reply by bryango** [reply]
I see... Thanks!
